MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl

```diff
@@ -1054,6 +1054,408 @@ resource configservice_label portal_config_manager_portal_labels_language {
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Cancel'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Annullér'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label portal_config_manager_portal_labels_export_notification_success_title {
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_EXPORT_NOTIFICATION_SUCCESS_TITLE'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Success'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Succes'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label portal_config_manager_portal_labels_export_notification_success_body {
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_EXPORT_NOTIFICATION_SUCCESS_BODY'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'The labels have been downloaded to your browser\'s download directory'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Mærkaterne er blevet downloadet til din browsers downloadmappe'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label portal_config_manager_portal_labels_export_notification_error_title {
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_EXPORT_NOTIFICATION_ERROR_TITLE'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Error'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Fejl'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label portal_config_manager_portal_labels_export_notification_error_body {
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_EXPORT_NOTIFICATION_ERROR_BODY'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Error exporting labels. Check your browser\'s console for more details.'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Fejl ved eksport af mærkaterne. Tjek din browsers konsol for flere detaljer.'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label portal_config_manager_portal_labels_import_dialog_title {
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_IMPORT_DIALOG_TITLE'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'WARNING: Overwriting labels'
```

Comment:
```suggestion
      default_translation = 'WARNING: Overwriting all {{language}} labels'
```

Reply:
Think that's too long, the language is in the body anyway